rpc/client.go

```diff
@@ -1,13 +1,14 @@
+// +build !js
```

Comment:
Same question. It doesn't look like much changed besides using `types` instead of `rpc`.

Reply:
This change is intentional. The `rpc` package should not be included in browser builds. The fact that it was included in the past was an oversight. Adding this build tag makes it so that it cannot be included. Without it, it's fairly easy to accidentally include it via transitive dependencies.

Reply:
Thanks for the answer, that makes sense 👍